recipe/meta.yaml

```
@@ -7,34 +7,50 @@ package:
 
 source:
   url: https://github.com/openforcefield/openff-toolkit/archive/{{ version }}.tar.gz
-  sha256: 2c94b79417d1217b15b68f40c09278d7c9e428e5527f928c8c29651ee07fbbb5
+  sha256: dc02048b46eb83abb7b1300cd979bc8ca52c8ad23c6f7de64a7af6d8135fe0da
 
 build:
-  noarch: python
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
-
-requirements:
-  host:
-    - python
-    - pip
-  run:
-    - python >=3.6
-    - numpy
-    - smirnoff99frosst
-    - openff-forcefields
-    - openmm
-    - networkx
-    - rdkit
-    - ambertools >=20
-    - mdtraj
-    - xmltodict
-  run_constrained:
-    - openforcefield ==9999999999
-
-test:
-  imports:
-    - openff.toolkit
+  number: 1
+
+outputs:
+  - name: openff-toolkit-base
+    script: build_base.sh
+    build:
+      noarch: python
+
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - python >=3.6
+        - numpy
+        - smirnoff99frosst
+        - openff-forcefields
+        - openmm
+        - networkx
+        - mdtraj
+        - xmltodict
+      run_constrained:
+        - openforcefield ==9999999999
+
+    test:
+      imports:
+        - openff.toolkit
+
+  - name: openff-toolkit
+    build:
+      noarch: python
+
+    requirements:
+      run:
+        - rdkit
+        - ambertools >=20
+        - {{ pin_subpackage('openff-toolkit-base', max_pin="x.x.x") }}
```

As per instructions [here](https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#transition-guide), I think we want this:

```suggestion
        - {{ pin_subpackage('openff-toolkit-base', exact=True) }}
```